internal/provider/resource_packer_channel_test.go

```diff
@@ -0,0 +1,249 @@
+			},
+		},
+	})
+}
+
+func TestAccPackerChannel_UpdateAssignedIteration(t *testing.T) {
+	resourceName := "hcp_packer_channel.production"
+	acctestChannelName := "update-test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t, map[string]bool{"aws": false, "azure": false}) },
+		ProviderFactories: providerFactories,
+		CheckDestroy: func(*terraform.State) error {
+			deleteBucket(t, acctestAlpineBucket, false)
+			return nil
+		},
+		Steps: []resource.TestStep{
+			{
+				PreConfig: func() {
+					fingerprint := "channel-update-it1"
+					upsertBucket(t, acctestAlpineBucket)
+					upsertIteration(t, acctestAlpineBucket, fingerprint)
+					itID, err := getIterationIDFromFingerPrint(t, acctestAlpineBucket, fingerprint)
+					if err != nil {
+						t.Fatal(err.Error())
+					}
+					upsertBuild(t, acctestAlpineBucket, fingerprint, itID)
+				},
+				Config: testConfig(testAccPackerChannelAssignedLatestIteration(acctestAlpineBucket, acctestChannelName)),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(resourceName, "author_id"),
+					resource.TestCheckResourceAttrSet(resourceName, "created_at"),
+					resource.TestCheckResourceAttrSet(resourceName, "id"),
+					resource.TestCheckResourceAttr(resourceName, "bucket_name", acctestAlpineBucket),
+					resource.TestCheckResourceAttr(resourceName, "name", acctestChannelName),
+					resource.TestCheckResourceAttrSet(resourceName, "iteration_assignment.0.id"),
+				),
+			},
+			{
+				PreConfig: func() {
+					fingerprint := "channel-update-it2"
+					upsertIteration(t, acctestAlpineBucket, fingerprint)
+					itID, err := getIterationIDFromFingerPrint(t, acctestAlpineBucket, fingerprint)
+					if err != nil {
+						t.Fatal(err.Error())
+					}
+					upsertBuild(t, acctestAlpineBucket, fingerprint, itID)
+				},
+				Config: testConfig(testAccPackerChannelAssignedLatestIteration(acctestAlpineBucket, acctestChannelName)),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(resourceName, "author_id"),
+					resource.TestCheckResourceAttr(resourceName, "bucket_name", acctestAlpineBucket),
+					resource.TestCheckResourceAttrSet(resourceName, "created_at"),
+					resource.TestCheckResourceAttrSet(resourceName, "id"),
+					resource.TestCheckResourceAttrSet(resourceName, "iteration_assignment.0.id"),
+					resource.TestCheckResourceAttrSet(resourceName, "iteration.0.id"),
+					resource.TestCheckResourceAttrSet(resourceName, "iteration.0.incremental_version"),
+					resource.TestCheckResourceAttr(resourceName, "iteration.0.fingerprint", "channel-update-it2"),
+					resource.TestCheckResourceAttr(resourceName, "name", acctestChannelName),
+					resource.TestCheckResourceAttrSet(resourceName, "updated_at"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccPackerChannel_UpdateAssignedIterationWithFingerprint(t *testing.T) {
+	resourceName := "hcp_packer_channel.staging"
+	acctestChannelName := "fingerprint-test"
```

Comment:
We've had some trouble with resource ID conflicts with our releaser acceptance test runs. Could you add a timestamp to this ID and the others? For example: https://github.com/hashicorp/terraform-provider-hcp/pull/440/files#diff-793cec3def4f921f6b01a086a16e6c0fb145f56b29613da7b36e9b10bcb4c922R17

Reply:
Thanks for this info. I'll update.